ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -51,18 +54,19 @@ function hasDuplicateSymbolAndDiffAddress(suggestedAssets, tokens) {
   return Boolean(duplicate);
 }
 
-const ConfirmAddSuggestedToken = (props) => {
-  const {
-    acceptWatchAsset,
-    history,
-    mostRecentOverviewPage,
-    rejectWatchAsset,
-    suggestedAssets,
-    tokens,
-  } = props;
-
+const ConfirmAddSuggestedToken = () => {
   const metricsEvent = useContext(MetaMetricsContext);
   const t = useContext(I18nContext);
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const mostRecentOverviewPage = useSelector((state) =>
+    getMostRecentOverviewPage(state),
+  );
```

Comment:
This can be simplified to this:
```suggestion
  const mostRecentOverviewPage = useSelector(getMostRecentOverviewPage);
  );
```